### PR TITLE
Placeholder ember-cli notes 14 September 2017

### PR DESCRIPTION
**Pull request by twokul**

### CODE REVIEW

File: ember-cli/2017-09/september-14.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# ember-cli Meeting Notes - 2017-09-14
+
+## Attendees
+
+
+## Agenda
```
Comment (twokul) [inline ember-cli/2017-09/september-14.md]: Please reply here with agenda items

Comment (twokul) [reply ember-cli/2017-09/september-14.md]: [Strategies RFC](https://github.com/ember-cli/rfcs/pull/110)

Comment (twokul) [reply ember-cli/2017-09/september-14.md]: https://github.com/ember-cli/ember-cli/pull/7331

Comment (rwjblue) [reply ember-cli/2017-09/september-14.md]: ES6 imports from `node_modules`

Comment (Turbo87) [reply ember-cli/2017-09/september-14.md]: https://github.com/ember-cli/ember-cli/pull/7232

**Comment by hjdivad** [comment]
superseded by #209

**Pull request closed without merging**